substrate-sgx/sp-io/Cargo.toml

```
@@ -21,15 +21,15 @@ sgx_types = { branch = "master", git = "https://github.com/apache/teaclave-sgx-s
 sgx-externalities = { default-features = false, path = "../externalities", optional = true }
 
 # Substrate dependencies
-sp-std = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24" }
-sp-core = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24", features=["full_crypto"] }
-sp-state-machine = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24", optional = true}
-sp-runtime-interface = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24" }
-sp-wasm-interface = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24" }
-sp-tracing = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24" }
-sp-trie = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24", optional = true }
-sp-keystore = {  default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24", optional = true }
-sp-externalities = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24", optional = true }
+sp-std = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26" }
+sp-core = { default-features = false, git = "https://github.com/paritytech/substrate.git", features=["full_crypto"] , branch = "polkadot-v0.9.26" }
+sp-state-machine = { git = "https://github.com/paritytech/substrate.git", optional = true, branch = "polkadot-v0.9.26" }
```

Most of the dependencies are already correctly defined, but it would be awesome if could do the following here:

```suggestion
sp-core = { default-features = false, features=["full_crypto"], git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26" }
sp-state-machine = { optional = true, git = "https://github.com/paritytech/substrate.git" branch = "polkadot-v0.9.26" }
/// transform the rest according to above pattern.
```

Then diener will not mess with it the next time, and we have the most critical part as the very first piece of information.